.github/workflows/cd.yaml

```diff
@@ -0,0 +1,15 @@
+name: "Continuous Deployment"
+on:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - published
+
+jobs:
+  cd:
+    uses: LCOGT/reusable-workflows/.github/workflows/continuous-deployment.yaml@main
+    with:
+      secretsExposedToSkaffold: 'FONTAWESOME_PACKAGE_TOKEN,'
```

Comment:
This allows us to send the package token to skaffold, which is used in the reusable workflow to build the docker image.